conf/modules/extra_dl.xml

```diff
@@ -11,7 +11,24 @@
   </header>
   <init fun="extra_pprz_dl_init()"/>
   <periodic fun="extra_pprz_dl_periodic()" freq="TELEMETRY_FREQUENCY" autorun="TRUE"/>
-  <event fun="ExtraDatalinkEvent()"/>
+
+  <!--
+  The event function listens on the extra telemetry port (from the payload computer for example)
+  and parses the received "datalink" messages, such as MOVE_WP.
+  We need this function to be able to listen to the extra telemetry port  
+
+  As a result all messages normally handled by the autopilot (such as MOVE_WP) are processed.
+  -->
+  <event fun="extra_pprz_dl_event()"/>
+
+  <!-- 
+  The "datalink" tag appends the specified "message" at the end of dl_parse_msg() function.
+  In case their is match in the message ID, "fun" is called on the message.
+
+  This automatically checks for the specified messages in regular telemetry (from GCS)
+  as well as on the extra telemetry (from the payload computer).
+  -->
+  <datalink message="PAYLOAD_COMMAND" fun="extra_pprz_dl_parse_payload_cmd()"/>
```

Comment:
I don't think this datalink event should be part of the module. `extra_dl` is only here to provide the IO stream for an extra telemetry/datalink channel. It is up to the module controlling your payload to implement this function. Also, another payload might want to reuse the same message but in a different way.